server/src/main/java/org/apache/iotdb/db/service/thrift/impl/DataNodeTSIServiceImpl.java

```diff
@@ -270,6 +270,11 @@ public TSExecuteStatementResp executeStatement(TSExecuteStatementReq req) {
         StatementGenerator.createStatement(
             statement, SESSION_MANAGER.getZoneId(req.getSessionId()));
 
+    TSStatus status = AuthorizerManager.getInstance().checkAuthority(s, req.sessionId);
+    if (status.getCode() != TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
+      return RpcUtils.getTSExecuteStatementResp(status);
+    }
+
```

Comment:
The analysis step has not been processed and the pathPattern in statement has not been parsed and the wildcard has not been removed.

Reply:
ok i will change it next